src/main/java/org/mockito/exceptions/base/MockitoInitializationException.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2007 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+
+package org.mockito.exceptions.base;
+
+public class MockitoInitializationException extends RuntimeException {
```

Comment:
I'm mentioning this, because as a public exception, `MockitoInitializationException` can be used by other framework and pissibly be misused.

Also this exception misses javadoc.